frontend/src/Components/Exercise.js

```diff
@@ -19,9 +20,22 @@ const Exercise = ({selected}) => {
         setInputVal(e.target.value)
     };
 
+    
+   
+
     const onTry = () => {
+        
         setOutputVal(inputVal);
-        setOutputAnswer(selected.code(...argSplit(inputVal)));
+
+        try {
+            const args = argSplit(selected.arg)
```

Comment:
arguments will be in a string divided by comas, so first we split them, then flatten